README.md

```diff
@@ -23,23 +23,28 @@ pip install personal-graph
 ### Building a Working Memory for an AI
 
 ```python
-from personal_graph import Graph
+from personal_graph.graph import Graph
+
+your-litellm-client = LLMClient(client="llm-client", model_name="llm-model-name")
+your-embedding-client = EmbeddingClient(client="embedding-client", model_name="embedding-model-name", dimensions="model-dimension")
 
-graph = Graph("your_db_url", "your_auth_token")
+graph = Graph("your_db_url", "your_auth_token", your-litellm-client, your-embedding-client)
 
 # Insert information into the graph
-graph.insert(text="Alice is Bob's sister. Bob works at Google.")
+graph.insert_into_graph(text="Alice is Bob's sister. Bob works at Google.")
```

Comment:
why this naming "insert_into_graph" - seems fairly verbose, to be discussed later

Reply:
we are already using insert() method so insert_into_graph() seems better choice to me as it relates with knowledge graph as well